x/ibc-transfer/client/cli/tx.go

```diff
@@ -92,6 +103,7 @@ to the counterparty channel. Any timeout set to 0 is disabled.`),
 		},
 	}
 
+	cmd.Flags().Uint64(flagTimeoutEpoch, 0, "Absolute timeout block epoch.")
```

Comment:
```suggestion
	cmd.Flags().Uint64(flagTimeoutEpoch, 0, "Absolute timeout block epoch. The timeout is disabled when set to 0.")
```

Reply:
Epoch starts at 0. Only `(0, 0)` means timeout height is disabled